lib/passport/passport.strategy.ts

```diff
@@ -5,7 +5,7 @@ export function PassportStrategy<T extends Type<any> = any>(
   Strategy: T,
   name?: string | undefined
 ): {
-  new (...args): InstanceType<T>;
+  new (...args): InstanceType<T> & { getPassportInstance(): passport.PassportStatic };
```

Comment:
`@types/passport` is not a required dependency. Isn't this change going to break projects where `skipLibCheck` is not set to true?

Reply:
yes

![image](https://github.com/nestjs/passport/assets/13461315/9c266c39-2926-45ab-a99f-f74d702834de)

We can work around that by adding  
`// @ts-expect-error TS2503: Cannot find namespace 'passport'`  
before that line